### PR TITLE
Dpos offline 2.0.0

### PR DESCRIPTION
**Pull request by vekexasia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,7 +85,9 @@
     "isomorphic-fetch": "^2.2.1",
     "pre-commit": "^1.2.2",
     "prettier": "^1.14.3",
-    "raf": "^3.4.0"
+    "raf": "^3.4.0",
+    "riw": "^0.8.0",
```
Comment (roosmaa) [inline package.json]: This one isn't a dependency anymore on `master` branch.
```suggestion
```

File: src/stores/ledger.ts [hunk 2]
```diff
@@ -339,17 +341,14 @@ class LedgerHub implements LedgerTaskRunner {
...
+          let signedTx: null | PostableRiseTransaction;
           transport.setExchangeTimeout(30000);
           try {
-            unsignedTx.signature = await comm.signTX(accountPath, txBytes, false);
-            signedTx = {
-              ...unsignedTx.toObj(),
-              senderId: account.address,
-            };
+            unsignedTx.signature = await comm.signTX(accountPath, txBytes, false) as Buffer & As<'signature'>;
+            signedTx = Rise.txs.toPostable(unsignedTx);
```
Comment (roosmaa) [inline src/stores/ledger.ts]: The _senderId_ field isn't needed anymore?

Comment (vekexasia) [reply src/stores/ledger.ts]: https://github.com/vekexasia/dpos-offline/blob/master/src/codecs/rise.ts#L26

Comment (roosmaa) [reply src/stores/ledger.ts]: And the resolve closure could be simplified as well then:

```
       const task = new LedgerTask(
         channelId,
         ({ account, signedTx }) => {
           // Add the account to the cache if we had cache-miss in the task runner
           if (!this.accountCache[accountSlot]) {
             this.accountCache[accountSlot] = account;
           }
           resolve(signedTx);
         },
         reject,
```

would become

```
       const task = new LedgerTask(
         channelId,
         resolve,
         reject,
```

**Pull request merged**